src/materialized/tests/pgwire.rs

```diff
@@ -246,13 +247,16 @@ fn test_persistence() -> Result<(), Box<dyn Error>> {
                 .query("SHOW INDEXES FROM mat", &[])?
                 .into_iter()
                 .map(|row| (row.get("Column_name"), row.get("Seq_in_index")))
-                .collect::<Vec<(String, i64)>>(),
-            &[
+                .collect::<HashSet<(String, i64)>>(),
+            [
```

Comment:
Minor Rust tip: if you use a `vec!` here, you won't need to call `.cloned`. (The fact that arrays don't allow you to iterate by value is one of my long standing annoyances with Rust. It's getting fixed soon though! https://github.com/rust-lang/rust/issues/25725)

Reply:
Done.